spec/features/saml/saml_spec.rb

```diff
@@ -199,6 +199,8 @@
         click_on 'Yes'
 
         complete_idv_profile_ok(user.reload)
+        click_submit_default
+        enter_correct_otp_code_for_user(user)
```

Comment:
The problem is with the proofer gem mock vendor. By only allowing one specific number that will allow the phone form to be submitted, we can no longer manually test the 2 scenarios. For example, by default our dev environment creates user `test1@test.com` with the 2FA phone `415-555-000`. If I want to test proofing while keeping that same number, I can't. In order to test keeping the same number, I have to cancel proofing, change my phone to `555-555-5555`, then go through proofing again. Then, if I want to test changing phones, I have to change it again back to something other than `555-555-5555`, then go through proofing again.

Instead of only allowing one specific value as the valid one, how about changing the gem so that there is only one possible invalid value?